CoffeeMachineSimulator/CoffeeMachineSimulator.UI/MainWindow.xaml.cs

```diff
@@ -12,7 +12,7 @@ public partial class MainWindow : Window
         public MainWindow()
```

Comment:
Please unstange this change, the connection string is your personal one, I need on the master branch on my repository the one that belongs to my subscription